db/migrate/20160830233154_add_port_ranges_to_load_balancer_listener.rb

```diff
@@ -0,0 +1,8 @@
+class AddPortRangesToLoadBalancerListener < ActiveRecord::Migration[5.0]
+  def change
+    change_table :load_balancer_listeners do |t|
+      t.column :load_balancer_port_range, :int4range, :default => nil, :null => true
+      t.column :instance_port_range, :int4range, :default => nil, :null => true
```

Comment:
Yeah, I am always wary of adding *more* Postgres specific stuff, only because it makes it that much harder to move away from it if we need to.  However, I'm good with this change. :+1: